stl/inc/format

```diff
@@ -39,9 +39,9 @@
 #define _FORMAT_
 #include <yvals_core.h>
 #if _STL_COMPILER_PREPROCESSOR
-#ifndef __cpp_lib_concepts
+#ifndef __cpp_lib_format
 #pragma message("The contents of <format> are available only with C++20 concepts support.")

```

Comment:
Should we adopt the error message?

It might be surprising for the user to find it not available in C++20

Reply:
I was hoping to get away without changing the message since this state is temporary, but that's not realistic given that "temporary" here means several months. Good catch.